apps/glusterfs/app_test.go

```diff
@@ -228,12 +230,14 @@ func TestAppBlockSettings(t *testing.T) {
 	defer os.Remove(dbfile)
 	os.Setenv("HEKETI_EXECUTOR", "mock")
 	defer os.Unsetenv("HEKETI_EXECUTOR")
+	os.Setenv("HEKETI_DB_PATH", dbfile)
+	defer os.Unsetenv("HEKETI_DB_PATH")
 
 	data := []byte(`{
 		"glusterfs" : {
 			"executor" : "crazyexec",
 			"allocator" : "simple",
-			"db" : "` + dbfile + `",
+			"db" : "/path/to/nonexistent/heketi.db",
```

Comment:
Same here: Why change the db settings at all if you're not testing them explicitly?

Reply:
Test case has added, please see below.